src/App.css

```diff
@@ -1,143 +1,115 @@
-App {
-  text-align: center;
-}
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+/* MY CSS*/
+.Paragraph {
+  font-size: small;
+  font-weight: bold;
+  color: white;
 }
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.hr {
+  border-bottom: 2px white;
 }
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.Todo {
   display: flex;
-  flex-direction: column;
+  margin: 0 -3rem 4px;
+  padding: 1.1rem 3rem;
+  justify-content: space-between;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-.App-link {
-  color: #61dafb;
+  background: rgba(75, 47, 47, 0.1);
 }
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.Todo-tittle {
+  margin: 0 -3rem 4px;
+  padding: 1.1rem 3rem;
```

Comment:
Use only one naming convention for all css class names. Kebab Case or Pascal case.
So name it like TodoTitle or todo-title